libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKWindowManager.h

```diff
@@ -87,7 +87,7 @@
  */
 @property (readonly,nonatomic,strong) SFSDKWindowContainer * _Nonnull snapshotWindow;
 
-/** SDK uses this window to present the passocde View.
+/** SDK uses this window to present the passcode View.
  */
 @property (readonly,nonatomic,strong) SFSDKWindowContainer * _Nonnull passcodeWindow;
```

Comment:
@bhariharan we originally talked about getting rid of these window properties in favor of the methods that take the nullable scene. Since we're not supporting passcode for the scene case in 9.0, should I keep the passcode one and get rid of the others?

Reply:
I think we would still go with the original plan for now, i.e. getting rid of the passcode window property as well. We can always deprecate the new method we create if needed.